apps/desktop/src/lib/utils/branch.test.ts

```diff
@@ -4,20 +4,30 @@ import { expect, test, describe } from 'vitest';
 describe.concurrent('getBranchNameFromRef', () => {
 	test('When provided a ref with a remote prefix, it returns the branch name', () => {
 		const ref = 'refs/remotes/origin/main';
+		const remote = 'origin';
 
-		expect(BranchUtils.getBranchNameFromRef(ref)).toBe('main');
+		expect(BranchUtils.getBranchNameFromRef(ref, remote)).toBe('main');
 	});
 
 	test('When provided a ref without a remote prefix, it returns the branch name', () => {
```

Comment:
This test name is no longer accurate

Reply:
Sorry about that! Fixed in the latest version